src/test/skript/tests/syntaxes/effects/EffSpread.sk

```diff
@@ -0,0 +1,77 @@
+test "EffSpread - 1":
```

Comment:
Needs positive tests with non-variable targets

Reply:
you mean spreading things across settable expressions besides variables?

Reply:
yes!